ui/components/app/confirm-hexdata/confirm-data.stories.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import README from './README.mdx';
+import ConfirmHexData from './confirm-hexdata';
+
+export default {
+  title: 'Components/App/ConfirmHexData',
+
+  component: ConfirmHexData,
+  parameters: {
+    docs: {
+      page: README,
+    },
+  },
+  argTypes: {
+    txData: 'object',
+  },
+};
+
+const txData = {
+  txParams: {
+    data: '0xa9059cbb000000000000000000000000b19ac54efa18cc3a14a5b821bfec73d284bf0c5e0000000000000000000000000000000000000000000000003782dace9d900000',
+    to: '0x0',
+  },
+  origin: 'https://metamask.github.io',
+  type: 'transfer',
+};
+
+const dataHexComponent = <div>Any custom component passed in props</div>;
+
+export const DefaultStory = (args) => {
+  return <ConfirmHexData txData={args.txData} />;
+};
+
+DefaultStory.storyName = 'Default';
+DefaultStory.args = {
+  txData,
+};
+
+export const DataHexComponentStory = (args) => {
+  return (
+    <ConfirmHexData
+      txData={args.txData}
+      dataHexComponent={args.dataHexComponent}
+    />
+  );
+};
+
+DataHexComponentStory.storyName = 'DataHexComponent';
+DataHexComponentStory.args = {
+  txData,
+  dataHexComponent,
+};
```

Comment:
Awesome! You could simplify this file slightly by utilizing spreading the args from the story object

```jsx
import React from 'react';
import README from './README.mdx';
import ConfirmHexData from './confirm-hexdata';

export default {
  title: 'Components/App/ConfirmHexData',
  component: ConfirmHexData,
  parameters: {
    docs: {
      page: README,
    },
  },
  argTypes: {
    txData: {
      control: 'object',
    },
    dataHexComponent: {
      control: 'element',
    },
  },
  args: {
    txParams: {
      data: '0xa9059cbb000000000000000000000000b19ac54efa18cc3a14a5b821bfec73d284bf0c5e0000000000000000000000000000000000000000000000003782dace9d900000',
      to: '0x0',
    },
    origin: 'https://metamask.github.io',
    type: 'transfer',
  },
};

export const DefaultStory = (args) => {
  return <ConfirmHexData {...args} />;
};

DefaultStory.storyName = 'Default';

export const DataHexComponent = (args) => {
  return <ConfirmHexData {...args} />;
};

DataHexComponent.args = {
  dataHexComponent: <div>Any custom component passed in props</div>,
};


```

Reply:
Done 👍